apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -49,31 +79,27 @@ class Courses extends Component {
       modernElementaryCoursesAvailable,
       specialAnnouncement
     } = this.props;
-    const headingText = isTeacher
-      ? i18n.coursesHeadingTeacher()
-      : i18n.coursesHeadingStudent();
-    const subHeadingText = i18n.coursesHeadingSubText({
-      linesCount: this.props.linesCount,
-      studentsCount: this.props.studentsCount
-    });
-    const headingDescription = isSignedOut
-      ? i18n.coursesHeadingDescription()
-      : null;
 
+    const {
+      headingText,
+      subHeadingText,
+      description,
+      buttonText
+    } = this.getHeroStrings();
     return (
       <div style={styles.content}>
         <HeaderBanner
```

Comment:
Note: I think we could _also_ change HeaderBanner to just not show the description if it's configured to show a short banner.